build.gradle

```diff
@@ -27,5 +27,8 @@ repositories {
 }
 
 dependencies {
+    jmh 'org.openjdk.jmh:jmh-core:1.20'
+    jmh 'org.openjdk.jmh:jmh-generator-bytecode:1.20'
+
```

Comment:
I think we should leave these dependencies out, since they're not runtime dependencies (or even test dependencies) and the jmh plugin seems to work fine even without them.

Reply:
OK, removed